terraform/applications/careers/static.tf

```diff
@@ -0,0 +1,225 @@
+# all the relevant infra for the static site
+resource "aws_s3_bucket" "logs" {
+  bucket = "mozilla-careers-logs-${local.environment}-783633885093"
+  acl    = "log-delivery-write"
+
+  force_destroy = false
+}
+
+resource "aws_s3_bucket" "mozilla-careers" {
+  bucket = local.bucket_name
+  acl    = "log-delivery-write"
+
+  force_destroy = false
+
+  hosted_zone_id = local.s3_zone
+  logging {
+    target_bucket = aws_s3_bucket.logs.bucket
+    target_prefix = "stage-logs/"
```

Comment:
I just tried to submit an edit for this to fix it to prod-logs, but you might have a different preference and/or I failed at that. Maybe ${local.environment} instead everywhere?